compatibility/v0/editorData.ts

```diff
@@ -0,0 +1,611 @@
+import { Schemas } from "$live/engine/schema/builder.ts";
```

Comment:
this file is supposedly used only for compatibility with the current admin version.

It is a bunch of functions copied from the live 0.x that serve as a "polyfill" for the admin as it is currently